### PR TITLE
First Capstone Project proposal

### PR DESCRIPTION
**Pull request by amon-cofie**
This is my first request for review on a working prototype of my first capstone project. In this Pull Request, I did the following things:

- Created two webpages: the homepage and the about page as requested.
- Added styles and behaviour with css and javascript 
- Added some animation effects with hover states on the pages
- Loaded the "featured speakers" section dynamically through javascript
- Implemented the function of the "show more" button on mobile screens.
- Deployed the working prototype on github pages: [Click here](https://amon-cofie.github.io/first-capstone-project/) to view the page
- This is a [loom video](https://www.loom.com/share/715079d714b5449f941e352dcbca9fbe) of a presentation on the project

### CODE REVIEW

**Review by emmiiorji — CHANGES_REQUESTED** [review]
Hi @amon-cofie, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project that has no descriptive README. Kindly include all main headers from the [README template](https://github.com/microverseinc/readme-template/blob/master/README.md) your repo.
Don't forget to include a link to a recorded video of maximum length 5 minutes where you present the features of the project.


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -2,14 +2,85 @@
...
+- Javascript
 
-<div> Icons made by <a href="https://www.flaticon.com/authors/freepik" title="Freepik"> Freepik </a> from <a href="https://www.flaticon.com/" title="Flaticon">www.flaticon.com'</a></div>
+## Live Demo
+
+
+![alt-text-1](./images/readMe-images/desktop-view.png "title-1") 
+![alt-text-2](./images/readMe-images/mobile-view.png "title-2")     ![alt-text-2](./images/readMe-images/mobile-gif.gif "title-2")
+![alt-text-1](./images/readMe-images/desktop-gif.gif "title-1")
+
```
Comment (ghost) [inline README.md:23]: - [ ] Kindly record a five-minute video using loom or any other tool presenting the features of the project you built and share the link to the video. This is for documentation purposes

File: index.html [hunk 2]
```diff
@@ -189,9 +193,11 @@ <h2>Partners</h2>
...
-        <div class="footer-logo">Logo from canva</div>
+        <div class="footer-logo">
+          <img src="./images/conference-logo.gif" alt="BMEN Global Logo" />
+        </div>
         <p class="copyright">
-          2015 Creative Commons Korea <br />
+          2022 BMEN Global Conference <br />
           Some Rights Reserved.
         </p>
       </div>
```
Comment (ghost) [inline index.html:203]: - [ ]  Kindly give your footer a dark background to match the original design

### Your footer

image

### Original Design
image

**Review by frank1738 — CHANGES_REQUESTED** [review]
Hi @amon-cofie , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

YesYouAreGIF (2)


## Highlights

✅ No linter error
✅ README is okay
✅ Good Design 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (amon-cofie) [reply index.html:203]: Thanks for the suggestion. I will make that change immediately

File: README.md (hunk 1 above)
Comment (amon-cofie) [reply README.md:23]: Okay. Thank you. I will do that immediately.

**Review by ichala — APPROVED** [review]
Hi @amon-cofie ,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  

 
68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966

# To Highlight 💪🏻

- Changes implemented✔️

 Good job implementing the required changes :+1: Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

*************************************
# Optional Suggestions 

- [ ] N/A


_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag  @ichala in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**